src/eventer/eventer_SSL_fd_opset.c

```diff
@@ -172,6 +172,7 @@ load_dh_params(const char *filename) {
   if(filename == NULL) return NULL;
   bio = BIO_new_file(filename, "r");
   if(bio == NULL) return NULL;
+  mtevL(mtev_notice, "Loading DH parameters from %s.\n", filename);
```

Comment:
So you would be OK with making the loading message `eventer_deb`?  If so, I'm happy to do that.